app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadFragment.kt

```diff
@@ -84,8 +97,13 @@ class SellPayloadFragment :
         SellOnlySolWarningBottomSheet.show(childFragmentManager)
     }
 
+    override fun navigateToInformationScreen() {
+        SellInformationBottomSheet.newInstance(childFragmentManager)
+    }
+
     override fun updateViewState(newState: SellPayloadContract.ViewState) = with(binding) {
         binding.widgetSendDetails.render(newState.widgetViewState)
+        widgetSendDetails.focusInputAndShowKeyboard()
```

Comment:
Are you sure you need to show a keyboard each time the ViewState is updated?

Reply:
need to show after shimmer hide, move to `override fun showLoading`